### ISSUE TITLE
Usability: Create a single bash script with that compiles and runs program

### ISSUE THREAD

**Issue by remyhunt** [acucciniello/DietCreator#1]
Hey Antonio, just a suggestions: put together a bash script that might help solve this, feel free to put this in code
I'm not sure of your application, but I would consider adding a `<script.sh>` to cut out the work for the user.

``` bash
echo Compiling...
g++ bulkingDiet.cpp cuttingDiet.cpp main.cpp -o run.o
echo -----------Ready-------------
./run.o
```

``` sh
$ chmod 700 script.sh
```

> this gives the script -rwx------ permissions so only you can run it (you can't run it by default)

``` sh
$ ./script.sh
```

![test](https://cloud.githubusercontent.com/assets/5217127/9892781/1ea0b914-5be0-11e5-834e-3f2893b1060a.png)

PS: Typo's in program and documentation - (Loose -> Lose)

**Comment by chinedufn** [comment]
@ashtonhunter but where's the PR at doe ?

**Comment by remyhunt** [comment]
@chinedufn gotta clean it up first!
tba: writing to the bash script with PHP w/ JQuery form input data. Precompiled on server -> injecting commands = cpp web application?

**Comment by remyhunt** [comment]
Resolved with pull request #2
